scripts/data/gen-nodetime/package.json

```
@@ -6,7 +6,7 @@
 		"nodetime": "./nodetime"
 	},
 	"scripts": {
-		"prebuild": "rm -rf dist && tsc",
+		"prebuild": "rm -rf dist",
```

```suggestion
		"prebuild": "rm -rf dist && tsc",
```

this one is for the nodetime cmd itself. js files after turned into a binary by vercel/pkg